notebooks/lerobot_act/act_to_openvino.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>I think we need an example to teach developer how to deploy this IR with OpenVINO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3111/discussion/#comment-2488270784'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Added evaluation section which shows how to run the model.